sqlglot/expressions.py

```diff
@@ -4116,20 +4140,31 @@ def to_column(sql_path: str | Column, **kwargs) -> Column:
     return Column(this=column_name, table=table_name, **kwargs)
 
 
-def alias_(expression, alias, table=False, dialect=None, quoted=None, **opts):
-    """
-    Create an Alias expression.
+def alias_(
+    expression: str | Expression,
+    alias: str | Identifier,
+    table: bool | t.Sequence[str | Identifier] = False,
+    quoted: bool | None = None,
+    dialect: t.Optional[DialectType] = None,
```

Comment:
`DialectType` contains `None` in the union already.

```suggestion
    dialect: DialectType = None,
```